src/CoreBundle/Services/AccountMailer.php

```diff
@@ -52,6 +52,29 @@ public function sendMailTo(Account $account, $subject, $template, array $templat
         $this->Mailer->send($message);
     }
 
+    /**
+     * @param Account $account
+     * @param $mailTo
+     * @param $subject
+     * @param $manualText
```

Comment:
`@param $subject` does not exist anymore, it's `$customText` now and their type (`string`) is missing.